### ISSUE TITLE
[Bug] Kirby and The Forgotten Land not launching from shortcut

### ISSUE THREAD

**Issue by nguyenvo96** [Ryujinx/Ryujinx#7069]
### Description of the issue

On the Steam deck game mode added as shortcut (non-steam game) and Desktop Mode as a shortcut added by Ryujinx to desktop, Kirby and the Forgotten Land does not load correctly anymore. On desktop mode, Ryujinx pops for a second before closing OR opens up Ryujinx to main screen with game selection. On Steam OS mode, it also either closes out or goes to the Ryujinx main screen as well.

This did not happen on Ryujinx version 1.1.1350

### Reproduction steps

Create a shortcut for Kirby and The Forgotten Land from Ryujinx main screen by right clicking the game and clicking on "create application shortcut". Attempt to start game by clicking on shortcut. Ryujinx will load to the game selection main screen without loading Kirby itself. 

### Log file

[Ryujinx_1.1.1354_2024-07-20_01-53-08.log](https://github.com/user-attachments/files/16319507/Ryujinx_1.1.1354_2024-07-20_01-53-08.log)

### OS

Steam OS 3.5.19

### Ryujinx version

1.1.1354

### Game version

1.0

### List of applied mods

None

**Comment by Jason-118** [comment]
I can't get any of my games to launch from shortcuts anymore on the Steam Deck, even if I create an shortcut from the application in desktop mode, it just shows the Ryujinx menu screen.

**Comment by Dutczar** [comment]
I had Tokyo Mirage Sessions crash a few days ago (after 30 hours of no issues) and the same issue started happening. I ignored it and assumed I overheated it, but this makes it seems like a general issue. I only tried reinstalling Ryujinx.

The difference is that in my case, launching from game mode usually results in closing after a second, but every 6 or so tries it manages to boot directly into the game, as opposed to Ryujinx' game selection like the poster mentioned.
